src/pages/community.tsx

```diff
@@ -226,6 +226,7 @@ const CommunityPage = () => {
       <PageMetadata
         title={t("page-community-meta-title")}
         description={t("page-community-meta-description")}
+        image="/heroes/community-hero.png"
```

Comment:
seems we already have a default image defined in `getOgImage` for `/community`

Reply:
Yeah, we could get rid of these where we already have the correct image defined in `imageForSlug`. Similar to other comment, tried to place these here for explicitness for each page, but I'm not married to that approach, both should work